jrpcfs/bypass_test.go

```diff
@@ -0,0 +1,64 @@
+package jrpcfs
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRpcBypass(t *testing.T) {
+	s := &Server{}
+
+	testBasicRpcBypass(t, s)
+}
+
+// TODO - move to API???
+func bypassMarshalReq(requestMethod string, ID uint64, tunnelReq interface{}) (bypassReq *BypassReq, err error) {
+
+	// Create request struct and convert tunnelReq to JSON
+	// TODO - need mount ID???
+	bypassReq = &BypassReq{
+		JSONrpc: "2.0",
+		Method:  requestMethod,
+		ID:      ID,
```

Comment:
Well this explains why you didn't need Params to be "json-annotated" (see earlier comment) :-)

BTW, I should have mentioned, there is actually an RpcPing API that would be most handy for your test code to actually exercise... I believe it responds with the # of bytes in whatever is sent to RpcPing()...